src/uu/mktemp/src/mktemp.rs

```diff
@@ -440,6 +440,7 @@ pub fn uu_app() -> Command {
                      may contain slashes, but mktemp creates only the final component",
                 )
                 .value_name("DIR")
+                .num_args(0..=1)
```

Comment:
I find this syntax a bit cryptic.
Could you please add a comment to explain what it means?
thanks

Reply:
As far as I understand, from (num_args)][https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.num_args], this allows to use an argument such as `--tmpdir` by itself. By adding `num_args(0..=1)` , it allows us to do `--tmpdir` by itself and not run into the error before where it thought no value was provided. 


So, something like
`Allows to use argument without a default value. Else, use provided input to generate tempdir` or something like that?

Let me know your thoughts and will add ti then to the code.

Reply:
yeah

tbh, I am also confused by the rust syntax 0..=1

Reply:
True, not the best syntax for that I'd agree. But, I have added comment in the code :)